packages/lesswrong/components/tagging/SubforumSubscribeSection.tsx

```diff
@@ -71,6 +71,7 @@ const SubforumSubscribeSection = ({
   
   const isSubscribed = currentUser?.profileTagIds?.includes(tag._id)
 
+  // TODO: Either reword or remove the tooltip here
```

Comment:
Is this a TODO for this PR?

Reply:
Actually I think we've settled on continuing to require membership to write discussion comments so the tooltip is accurate again, I've removed the TODO